ClinGen/clincoded
# Connects to #1772. New users and affiliations.

**Steps to test:**
1. Go to ^/users and enter `Lückerath` or `Baxter` into the **Filter table by** text field.
2. Expect to see an user record being returned for either `Katharina Lückerath` and `Samantha Baxter`.
